docs/migration-guides/1.16-1.17.md

```diff
@@ -0,0 +1,38 @@
+# Migration from v1.16 to v1.17
+
+## Eventing
+
+In Release 1.17 we remove the support for Knative-Serving from kyma. After the upgrade Knative-Serving is still 
+installed and fully functional in the cluster, but can be removed easily in order to free resources. Removing 
+Knative-Serving will not affect supported kyma functionality.
+
+Before you remove Knative-Serving from the cluster make sure that you take a backup of any of your own resources that used 
+Knative-Serving. You can do this by running:
+
+```bash
+kubectl get serving -A -oyaml > <some-backup-file>
```

Comment:
```suggestion
kubectl get serving -A -oyaml > <backup-file-name>
```

Maybe it's just me, but `some backup file` sounds a little too casual. Hopefully `backup-file-name` is also clear